Point_set_processing_3/include/CGAL/OpenGR/compute_registration_transformation.h

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) 2018  GeometryFactory(France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+// You can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0+
+//
+// Author(s) : Sebastien Loriot
+
+#ifndef CGAL_OPENGR_COMPUTE_REGISTRATION_TRANSFORMATION_H
```

Comment:
This file needs an `#ifdef CGAL_OPEN_GR_ENABLED` otherwise travis individual header compilation will fail.